### DISCUSSION TITLE
Promote your server!

### DISCUSSION THREAD

**Discussion by matsn0w** [matsn0w/IIS-EmergencyLS-ELS-FiveM#43] [Show and tell]
Do you use MISS ELS on your server? Then you can promote it here! Also don't forget to showcase your patterns in [this topic](https://github.com/matsn0w/MISS-ELS/discussions/42).

A few simple rules:

* you can only post **exactly one** promotion
* you must have MISS ELS installed and enabled on your server
* be kind to each other

You are free to:

* link to your Discord server or website
* attach an image to your post
* invite people to join your server

I may delete your post if I don't think it fits this topic.

**Comment by DavleyNL** [comment]
I am proud to say that we were the first server with MISS-ELS installed, due to that, we managed to work closely with Matsn0w to implement and test some stuff!

We are DavleyRP, a Dutch Roleplay clan focussed on Emergency Services!
We have two law enforcements services; Police & Royal Military Police (Koninklijke Marechaussee)
To provide support we have a few other services like Ambulance, Fire Department, Traffic Management (Rijkswaterstaat), Royal Dutch Rescue Services (KNRM) and a Tow service (Bergnet)

Together with our experienced civilians we re-create realistic scenario's in-game that our emergency services can take part in.
Due to in-house experience about emergency services and roleplay we manage to keep our reports/scenario's creative and new!

For more information, check out our Discord server:
https://discord.gg/9BPBytF5ym
